### PR TITLE
Moved checkout above setup step to align with the other samples

### PR DESCRIPTION
**Pull request by ThomasArdal**
To align the second code sample with the other samples, I have moved the checkout step above the install step.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ Multiple versions:
 
 ```yml
 steps:
-- name: Setup dotnet
 - uses: actions/checkout@v2
+- name: Setup dotnet
 - uses: actions/setup-dotnet@v1
```
Comment (MaksimZhukov) [inline README.md:39]:
```suggestion
  uses: actions/setup-dotnet@v1
```

Comment (ThomasArdal) [reply README.md:39]: Not sure I understand this suggestion. What did you change? 😄

Comment (ThomasArdal) [reply README.md:39]: Ah, you removed the `-`. Good catch.

**Review by MaksimZhukov — APPROVED** [review]

**Comment by vsafonkin** [comment]
Hi @ThomasArdal, could you update your branch from main? This issue was fixed #258

**Comment by ThomasArdal** [comment]
@vsafonkin I think that I did that now.

**Review by vsafonkin — APPROVED** [review]

**Comment by MaksimZhukov** [comment]
@ThomasArdal Thank you for the contribution!

**Pull request merged**